### ISSUE TITLE
Element not found by front end highlighting: aria-hidden icon in the Instagram Feed Pro Developer plugin

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#102]
On https://amberhinds.com/about/

This element cannot be found by Accessibility Checker:
`<svg class="svg-inline--fa fa-clock fa-w-16" aria-hidden="true" data-fa-processed="" data-prefix="far" data-icon="clock" role="presentation" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 512 512"><path fill="currentColor" d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm0 448c-110.5 0-200-89.5-200-200S145.5 56 256 56s200 89.5 200 200-89.5 200-200 200zm61.8-104.4l-84.9-61.7c-3.1-2.3-4.9-5.9-4.9-9.7V116c0-6.6 5.4-12 12-12h32c6.6 0 12 5.4 12 12v141.7l66.8 48.6c5.4 3.9 6.5 11.4 2.6 16.8L334.6 349c-3.9 5.3-11.4 6.5-16.8 2.6z"></path></svg>`

It's the clock icon that is only visible when you hover over a picture in the Instagram feed.

<img width="476" alt="Screen Shot 2023-02-09 at 1 04 22 PM" src="https://user-images.githubusercontent.com/13153456/217912612-9d95eaec-46df-4df4-bfdb-59b90242e311.png">

**Comment by SteveJonesDev** [comment]
@amberhinds, can you provide me this URL with the issue ID parameter?

**Comment by SteveJonesDev** [comment]
The plugin is now finding this.

![Screen Shot 2023-04-07 at 4 41 52 PM](https://user-images.githubusercontent.com/2895788/230676232-e2c54c1e-e1e4-4005-be08-920519f99539.png)
